LibTrixi.jl/src/api_c.jl

```diff
@@ -461,6 +461,53 @@ trixi_load_primitive_vars_cfptr() =
     @cfunction(trixi_load_primitive_vars, Cvoid, (Cint, Cint, Ptr{Cdouble}))
 
 
+"""
+    trixi_store_in_database(data::Ptr{Cdouble}, size::Cint, index::Cint,
+                            simstate_handle::Cint)::Cvoid
+
+Store data vector in current simulation's database.
+
+A reference to the passed data array `data` will be stored in the database of the simulation
+given by `simstate_handle` at given `index`. The database object has to be created in
+`init_simstate()` of the running libelixir and can be used throughout the simulation.
+
+The database object has to exist, has to be of type `LibTrixiDataBaseType`, and has to hold
+enough data references such that access at `index` is valid.
+
+The size of `data` has to match `size`.
```

Comment:
IMHO it would be good to emphasize here that the memory storage remains on the user side and that whenever the underlying object is freed, it should also be removed from the database

Reply:
Done!